### ISSUE TITLE
Unable to connect to local area network

### ISSUE THREAD

**Issue by Kino0208** [MrKacafirekCZ/doggomod#12]
After being connected to the local area network for a period of time, I was suddenly kicked out by the server, and then I can't enter the connection again.
My guess is that when the block of the wolf was touched, what did the wolf do, and then caused the kick.
I translated this sentence with translation software, I can't speak English, sorry.
Chinese :
局域网联机一段时间后 ，突然被服务器踢出，之后再也无法进入联机
我的猜测是，碰到了狼的区块，狼做了什么动作，然后导致的踢出。
这段话是我用翻译软件翻译的，我不会英语 对不起
[log.txt.txt](https://github.com/MrKacafirekCZ/doggomod/files/8969063/log.txt.txt)
.

**Comment by MrKacafirekCZ** [comment]
Hello, it seems like you're running a lot of mods and it could be that there is a mod that isn't compatible with my Doggo mod and therefore it's causing the game to crash. May I see your full list of mods that you're using?

(I would also like to remind you that currently Doggo mod has issues when running in multiplayer or in local area network servers. Therefore there is a possibility this could be the issue as well.)

**Comment by Kino0208** [comment]
Of course, at present, I have removed this mod. That day, I used two computers to alternate files repeatedly, remove mods in batches, start the game repeatedly, and conduct online tests. After six hours, the result I ruled out was this Doggo mod
![mods list](https://user-images.githubusercontent.com/76459545/175458560-ed05338b-4d68-4bf0-9526-53a469e673f3.PNG)
.

**Comment by MrKacafirekCZ** [comment]
Unfortunately I don't think my Doggo mod will work with this many mods. My mod is changing the fundamentals of how wolfs and dogs work in Minecraft therefore there is a good chance that more than one mod is colliding with mine. Unfortunately I don't know about most of these mods so I won't be able to say which mods are the ones that are causing Doggo mod not to function.
